x/pricefeed/client/rest/tx.go

```diff
@@ -5,23 +5,26 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/gorilla/mux"
-
 	"github.com/cosmos/cosmos-sdk/client/context"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/rest"
 	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
-	"github.com/kava-labs/kava/x/pricefeed/types"
+	"github.com/gorilla/mux"
+
+	// 	authclient "github.com/cosmos/cosmos-sdk/x/auth/client" TODO: use instead of utils?
```

Comment:
remove TODO, `utils` is correct import

Reply:
https://github.com/Kava-Labs/kava/pull/359/commits/d6df1fb7305aec68497e06464e15f6ae634eb619